gammapy/maps/wcsnd.py

```diff
@@ -102,7 +104,14 @@ def from_hdu(cls, hdu, hdu_bands=None):
         shape_wcs = tuple([np.max(geom.npix[0]),
                            np.max(geom.npix[1])])
         meta = cls._get_meta_from_header(hdu.header)
-        map_out = cls(geom, meta=meta)
+
+        # Read unit
```

Comment:
Remove comment "read unit".
Comments that just say the same thing as the code are useless, even harmful, because 2x as many lines and very hard to keep comments updated.
So IMO we should strive for clean code and only add comments where they add extra value over the code.

Reply:
OK, indeed this one is pretty obvious